DQMServices/Core/interface/DQMEDHarvester.h

```diff
@@ -24,13 +26,14 @@ class DQMEDHarvester
   // implicit assignment operator
   // implicit destructor
   virtual void beginRun(edm::Run const&, edm::EventSetup const&) {};
-  virtual void analyze(edm::Event const&, edm::EventSetup const&) final {};
+  virtual void produce(edm::Event&, edm::EventSetup const&) override final {};
```

Comment:
`endLuminosityBlock` is still needed since it calls `dqmEndLuminosityBlock`. `beginLuminsityBlock()` needs to be defined since marking it `final` keeps people who inherit from `EDHarverster` from overriding that function and therefore changing the expected behavior of this class.